### PR TITLE
chore: Update examples to use latest ALB module

### LINKED ISSUE #129: docs: Incorrect default in description of `desired_count`
In the `service` module, the variable description below is incorrect considering the default value.

Note, "Defaults to 0" could probably be removed entirely, as the default value is displayed in Terraform docs.

```
variable "desired_count" {
  description = "Number of instances of the task definition to place and keep running. Defaults to `0`"
  type        = number
  default     = 1
}
```

### PR DESCRIPTION
**Pull request by bryantbiggs**
## Description
- Update examples to use latest ALB module
- Fix variable description for default number of tasks

## Motivation and Context
- Resolves #129 

## Breaking Changes
- No

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: examples/complete/main.tf [hunk 1]
```diff
@@ -188,51 +188,72 @@ resource "aws_service_discovery_http_namespace" "this" {
...
 
-  load_balancer_type = "application"
+      forward = {
+        target_group_key = "ex-ecs"
+      }
+    }
+  }
+
+  target_groups = {
+    ex-ecs = {
```
Comment (antonbabenko) [inline examples/complete/main.tf:233]: Let's not use `-` in keys. I thought `tflint` complains about such already.

File: examples/fargate/main.tf [hunk 2]
```diff
@@ -171,51 +171,72 @@ resource "aws_service_discovery_http_namespace" "this" {
...
-
-  target_groups = [
-    {
-      name             = "${local.name}-${local.container_name}"
-      backend_protocol = "HTTP"
-      backend_port     = local.container_port
-      target_type      = "ip"
-    },
-  ]
+      # Theres nothing to attach here in this definition. Instead,
```
Comment (antonbabenko) [inline examples/fargate/main.tf:235]:
```suggestion
      # There's nothing to attach here in this definition. Instead,
```

**Review by antonbabenko — APPROVED** [review]
LGTM in general

File: examples/complete/main.tf (hunk 1 above)
Comment (bryantbiggs) [reply examples/complete/main.tf]: got it! Updated in https://github.com/terraform-aws-modules/terraform-aws-ecs/pull/130/commits/c69ba10adb76fe1f95d15054a613014204ad6b4c

**Comment by antonbabenko** [comment]
This PR is included in version 5.6.0 :tada:

**Pull request merged**